virttest/guest_agent.py

```diff
@@ -822,5 +825,36 @@ def guest_file_seek(self, handle, offset, whence):
                        2,file end
         :return: a dict with position and eof.
         """
-        return self._guest_file_operation("seek", handle=handle, offset=offset,
-                                          whence=whence)
+        cmd = "guest-file-seek"
+        return self._cmd_args_update(cmd, handle=handle, offset=offset,
+                                     whence=whence)
+
+    def guest_exec(self, path, arg=None, env=None, input_data=None,
+                   capture_output=None):
```

Comment:
`capture_output=False`

Reply:
No, if capture_output =None, guest-exec cmd will use default value(false)

Reply:
if capture_output =None, `_cmd_args_update() ` function will handle it to delete this key, then guest-exec cmd will use the default value.
such as:  {"execute": "guest-exec", "arguments": {"path": "echo", "arg": ["This", "is", "a", "test."]}}
this qga cmd will adopt the default `capture_output` value.